core/cloudflow-operator/src/main/scala/cloudflow/operator/runner/SparkRunner.scala

```diff
@@ -257,9 +400,11 @@ object SparkResource {
   final case class Executor(
       instances: Int,
       cores: Option[Int] = Some(1),
+      coreRequest: Option[String] = None,
       coreLimit: Option[String] = None,
       memory: Option[String] = None,
       memoryOverhead: Option[String] = None,
+      env: Option[List[EnvVar]] = None,
```

Comment:
Plural here too

Reply:
This is the mapping to the CR so can't just be changed.

Reply:
Same